pandas/tests/reductions/test_reductions.py

```diff
@@ -689,6 +689,16 @@ def test_ops_consistency_on_empty(self, method):
         result = getattr(Series(dtype=float), method)()
         assert pd.isna(result)
 
+        # Nullable dtype on mean
+        eser = Series([], dtype="Int64")
+        nser = Series([np.nan], dtype="Int64")
+        if method == "mean":
+            result = getattr(eser, method)()
+            assert result is pd.NA
+
```

Comment:
can we parameterize this test on the method and dtype (you can also move var to a new function if that is easier)

Reply:
so a new function parameterized on method "mean" and dtype - all nullable dtypes?